examples/0-abstraction-hh-rlhf/rl_finetuning.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from datasets import load_dataset
+from peft import LoraConfig
+from tqdm import tqdm
+from transformers import LlamaTokenizer
+
+from trl import AutoModelForCausalLMWithValueHead, PPOConfig, PPOTrainer
+from trl.core import LengthSampler
+
+
+rm_adapter_id = "ybelkada/llama-7b-dummy-rm"
```

Comment:
Have you run an experiment just using sentiment analysis (i.e. train the RM on IMDB) as a sanity check everything works?